src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -0,0 +1,1108 @@
+namespace System.Security {
+  public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk {
```

Comment:
Should we be doing things like turning off EditorBrowsable? If possible, marking lots of things here Obsolete would be good too since we really don't want people to start using these for security.